mlir/lib/Dialect/SCF/IR/SCF.cpp

```diff
@@ -1415,6 +1415,19 @@ InParallelOp ForallOp::getTerminator() {
   return cast<InParallelOp>(getBody()->getTerminator());
 }
 
+SmallVector<Operation *> ForallOp::getStoreOpUser(BlockArgument bbArg) {
+  SmallVector<Operation *> storeOps;
+  InParallelOp inParallelOp = getTerminator();
+  for (Operation &yieldOp : inParallelOp.getYieldingOps()) {
```

Comment:
I was hoping that `ParallelCombiningOpInterface` is implemented by the ops in the "combining" region (e.g., `parallel_insert_slice`), but its actually implemented by the op that has the region (`in_parallel`). Then we wouldn't have to hard-code `parallel_insert_slice` here. But nvm, we can fix that later...